source/apiVolontaria/apiVolontaria/settings.py

```diff
@@ -213,6 +216,18 @@
     }
 }
 
+ANYMAIL = {
+    "SENDINBLUE_API_KEY": "example_api_key",
+}
+
+SENDINBLUE_TEMPLATES = {
+    "CALENDAR_INVITATION": "example_template_id",
+}
+
+EMAIL_BACKEND = "anymail.backends.sendinblue.EmailBackend"
+DEFAULT_FROM_EMAIL = "benevoles@nousrire.com"
```

Comment:
please try to let `default` value as generic as possible. Project can be use by other project and not necessary by `nousrire`.

It can be dangerous because NousRire can forget to change this value and if we change the default value in the future their process of deployment can fail.

Reply:
Oups !  Good catch !